uelc/main/models.py

```diff
@@ -566,4 +566,40 @@ def unlocked(self, user, section):
                 unlocked = True
         return unlocked
 
+
+class CaseAnswer(models.Model):
+    answer = models.ForeignKey(Answer)
+    title = models.CharField(max_length=255, blank=True)
```

Comment:
If you have a `CharField` with a `max_length`, you need to deal with that in the validation layer to avoid 500s. My recommendation is generally to always prefer `TextField` to `CharField` since it doesn't come with landmines.